MagnetScannerIOS/NetworkResolver.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NetworkResolver.swift
+//  Magnet
+//
+//  Created by Francisco Jordano on 20/10/2016.
+//
+
+import Foundation
+import Alamofire
+import SwiftyJSON
+
+//
+// In charge of fetching the urls associated to the latitude and longitude
+// passed.
+//
+public class NetworkResolver {
+  static let API_END_POINT: String = "https://tengam.org/content/v1/search/beacons/"
+  static let SLUG_BASE_URL: String = "https://tengam.org/"
+  static let RADIUS: Int = 1500
+  
+  static func resolveLocation(lat: Double, lon: Double, callback: (Array<JSON> -> Void)!) {
+    let url: String = "\(API_END_POINT)\(lat),\(lon),\(RADIUS)"
+    
+    Alamofire.request(.GET, url).responseJSON { response in
+      let empty: Array<JSON> = Array<JSON>()
+      guard response.result.isSuccess else {
+        callback(empty)
+        return
+      }
+      
+      if let json: JSON = JSON(response.result.value!) {
+        debugPrint("JSON: \(json)")
+        callback(filterJSON(json))
+      } else {
+        callback(empty)
+      }
+    }
+  }
+  
+  private static func filterJSON(json: JSON) -> Array<JSON> {
```

Comment:
Right now is just a way of removing the extra json fields that the url returns in order to return just {'url': 'xxxxx'} as the rest of scanners do.